useragent.gemspec

```diff
@@ -1,17 +1,18 @@
 Gem::Specification.new do |s|
-  s.name    = "useragent"
-  s.version = "0.16.11.pre.3b5e242"
+  s.name    = 'useragent'
+  s.version = '0.16.11.pre.3b5e242'
 
-  s.homepage    = "https://github.com/gshutler/useragent"
-  s.summary     = "HTTP User Agent parser"
-  s.description = "HTTP User Agent parser"
+  s.homepage    = 'https://github.com/gshutler/useragent'
+  s.summary     = 'HTTP User Agent parser'
+  s.description = 'HTTP User Agent parser'
 
-  s.files = Dir["LICENSE", "README.md", "lib/**/*.rb"]
+  s.files = Dir['LICENSE', 'README.md', 'lib/**/*.rb']
```

Comment:
```suggestion
  s.files = Dir['LICENSE', 'CHANGELOG.md', 'README.md', 'lib/**/*.rb']
```

Reply:
Also we should probably keep the changelog up to date since another contributor added it